src/dotnet-compile-fsc/Program.cs

```diff
@@ -168,11 +174,8 @@ public static int Run(string[] args)
                 allArgs.Add("--target:exe");
 
                 //HACK we need default.win32manifest for exe
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-                {
-                    var win32manifestPath = Path.Combine(AppContext.BaseDirectory, "default.win32manifest");
-                    allArgs.Add($"--win32manifest:{win32manifestPath}");
-                }
+                var win32manifestPath = Path.Combine(fscExeDir, "..", "..", "runtimes", "any", "native", "default.win32manifest");
```

Comment:
thx @brthor , i see the same problem with rel/1.0.0 HEAD
I added https://github.com/Microsoft/visualfsharp/issues/1024 to ask to embedd it, so in the future we can remove it (relative paths may change, for example contentFiles instead of runtime/any )

Yesterday i tried this pr, but i was too slow